tests/kernel/mp/prj.conf

```diff
@@ -1,2 +1,2 @@
 CONFIG_ZTEST=y
-CONFIG_SMP=n
+CONFIG_MP_NUM_CPUS=1
```

Comment:
This looks quite odd as in the test itself we check for `CONFIG_MP_NUM_CPUS > 1`:

```
BUILD_ASSERT(CONFIG_MP_NUM_CPUS > 1);
```

https://github.com/zephyrproject-rtos/zephyr/blob/master/tests/kernel/mp/src/main.c#L16